enterprise/fulltext-addon/src/main/java/org/neo4j/kernel/api/impl/fulltext/integrations/bloom/BloomKernelExtensionFactory.java

```diff
@@ -41,7 +41,7 @@
 public class BloomKernelExtensionFactory extends KernelExtensionFactory<BloomKernelExtensionFactory.Dependencies>
```

Comment:
Other factories have `@Service.Implementation( KernelExtensionFactory.class )`, not sure if we should adopt such black magic though...